operators/community-windows-machine-config-operator/5.1.0/manifests/community-windows-machine-config-operator.5.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,355 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
+    certified: "false"
+    createdAt: "2022-06-01T09:47:21Z"
+    description: An operator that enables Windows container workloads on OCP
+    olm.skipRange: '>=4.0.0 <5.0.0'
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
+    operators.openshift.io/valid-subscription: '["Red Hat OpenShift support for Windows Containers"]'
```

Comment:
Should this line should be removed from the community CSV?

Reply:
yes, will remove